src/services/event.ts

```diff
@@ -105,6 +117,48 @@ async function findEventHelper(where: WhereOptions, plural = false): Promise<Eve
   };
 }
 
+interface FindEventHelperBlobOptions {
+  key: string;
+  value: string;
+  regions: number[] | undefined;
+}
+
+async function findEventHelperBlob({ key, value, regions }: FindEventHelperBlobOptions): Promise<EventShape[]> {
+  // let whereClause = `data->>'${key}' = '${value}' OR NOT (data ? '${key}')`;
+  // if (regions && regions.length) {
+  //   whereClause += ` AND regionId IN (${regions.join(',')})`;
+  // }
+
+  const where = {
+    data: {
+      [Op.or]: [
+        { [key]: value },
+        { [key]: { [Op.eq]: null } },
```

Comment:
Not sure if the eq NULL should be controlled by a param. Because this is a generic function we might get back in other scenarios we are not expecting.

Reply:
Can we say if NULL then default to some value?